frontend/src/components/mine/Tailings/MineTailingsInfo.js

```diff
@@ -88,22 +102,36 @@ export class MineTailingsInfo extends Component {
     updatedDocument.due_date = value.tsf_report_due_date;
     updatedDocument.received_date = value.tsf_report_received_date;
     updatedDocument.exp_document_status_guid = value.tsf_report_status;
-    this.props.updateExpectedDocument(updatedDocument.exp_document_guid, {document: updatedDocument}).then(() => {
-      this.props.closeModal();
-      this.props.fetchMineRecordById(this.props.mine.guid);
-    })
+    this.props
+      .updateExpectedDocument(updatedDocument.exp_document_guid, { document: updatedDocument })
+      .then(() => {
+        this.props.closeModal();
+        this.props.fetchMineRecordById(this.props.mine.guid);
+      });
   };
 
   openEditReportModal(event, onSubmit, title, statusOptions, doc) {
     this.setState({
-      selectedDocument: doc
-    })
+      selectedDocument: doc,
+    });
     event.preventDefault();
     const initialValues = {
-      "tsf_report_name": doc ? ((doc.exp_document_name === 'None') ? null : doc.exp_document_name) : null,
-      "tsf_report_due_date": doc ? ((doc.due_date === 'None') ? null : doc.due_date) : null,
-      "tsf_report_received_date": doc ? ((doc.received_date === 'None') ? null : doc.received_date) : null,
-      "tsf_report_status": doc ? ((doc.exp_document_status_guid === 'None') ? null : doc.exp_document_status_guid) : null,
+      tsf_report_name: doc
```

Comment:
Nested ternaries concern me. We might be able to do something like this:
```
tsf_report_name: (doc && doc.exp_document_name !== "None") ? doc.exp_document_name : null
```

Not necessary for this PR,.